.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
📝 This change ensures line end-of-line normalization does not cause behavior differences between users on Windows and users on other platforms.